go.mod

```diff
@@ -46,14 +46,14 @@ require (
 	github.com/ipfs/go-log v1.0.5
 	github.com/ipfs/go-log/v2 v2.5.1
 	github.com/libp2p/go-addr-util v0.1.0
-	github.com/libp2p/go-libp2p v0.23.3
+	github.com/libp2p/go-libp2p v0.24.0
```

Comment:
Is there any breaking change to us?

Reply:
A couple. Mostly tests and transient deps.